MediaBrowser.Common/Json/Converters/JsonVersionConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace MediaBrowser.Common.Json.Converters
+{
+    /// <summary>
+    /// Converts a Version object or value to/from JSON.
+    /// </summary>
+    /// <remarks>
+    /// Required to send <see cref="Version"/> as a string instead of an object.
+    /// </remarks>
+    public class JsonVersionConverter : JsonConverter<Version>
+    {
+        /// <inheritdoc />
+        public override Version Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+            => new Version(reader.GetString());
+
+        /// <inheritdoc />
+        public override void Write(Utf8JsonWriter writer, Version value, JsonSerializerOptions options)
+            => writer.WriteStringValue(value.ToString());
```

Comment:
Could possibly prevent a string alloc here by doing a stack alloc and using `Version.TryFormat`